ci/docker/os-image-stemcell-builder/Dockerfile

```
@@ -1,126 +1,124 @@
+    && cd - \
+    && rm -rf ruby-install-* \
+    && echo 'gem: --no-document' > /etc/gemrc \
+    && NUM_CPUS=$(grep -c ^processor /proc/cpuinfo) \
+    && ruby-install --jobs=${NUM_CPUS} --cleanup --system ruby ${RUBY_VERSION} \
+      -- --disable-install-doc --disable-install-rdoc \
+    && gem update --system \
+    && bundle config --global path "${GEM_HOME}" \
+    && bundle config --global bin "${GEM_HOME}/bin"
+ENV PATH=${GEM_HOME}/bin:${PATH}
+
+RUN syft_cli_path="/usr/local/bin/syft" \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)